docs/source/model_doc/lxmert.rst

```diff
@@ -52,7 +52,8 @@ Tips:
   contains self-attention for each respective modality and cross-attention, only the cross attention is returned and
   both self attention outputs are disregarded.
 
-The original code can be found `here <https://github.com/airsplay/lxmert>`__.
+This model was contributed by `lysandre <https://huggingface.co/eltoto1219>`__. The original code can be found `here
```

Comment:
It's nice to credit me but it's actually @eltoto1219's doing :) Antonio, do you have a HF account so that we may credit you accordingly?

Reply:
Oh I forgot to properly paste on the second half ;-) It's https://huggingface.co/eltoto1219 I think.